frontend/src/lib/components/ResizableTable/ResizableTable.tsx

```diff
@@ -13,7 +13,7 @@ import './index.scss'
 
 export const TableConfig = _TableConfig
 
-export interface ResizableColumnType<RecordType> extends ColumnType<RecordType> {
+export interface ResizableColumnType<RecordType> extends Omit<ColumnType<RecordType>, 'width'> {
```

Comment:
`width` property is set dynamically internally, therefore it shouldn't be passed.